client/my-sites/site-settings/media-settings.jsx

```diff
@@ -72,10 +91,51 @@ class MediaSettings extends Component {
 					label={ translate( 'Enable fast, ad-free video hosting' ) }
 					disabled={ isRequestingOrSaving }
 				/>
+				{ this.props.isVideoPressActive && this.renderVideoStorageIndicator() }
 			</FormFieldset>
 		);
 	}
 
+	renderVideoStorageIndicator() {
+		const {
+			mediaStorage,
+			siteId,
+			sitePlanSlug,
+			siteSlug,
+			translate,
+		} = this.props;
+
+		// The API may use -1 for both values to indicate special cases
+		const isStorageDataValid = mediaStorage.storage_used_bytes > -1;
```

Comment:
This is an interesting area where I'm not sure what the best approach is. We have plans data, and we can check unlimited storage based on plans `hasFeature` selector. I explored this approach in #16806 [here](https://github.com/Automattic/wp-calypso/blob/a822f9ecac151329b2404374a8234404f20c5d0a/client/blocks/video-storage/index.js#L77-L80).

As I consider this, dedicated selectors for each property of `mediaStorage` is actually rather appealing. It would flatten the props of this component nicely and is probably worth including in this PR. I'm going to hold off until [conversation around a `VideoStorage` component](https://github.com/Automattic/wp-calypso/pull/14229#discussion_r119158725) has concluded.